<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" lang="en">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
		<title>ISMICT 2024 || Program</title>
		<link href="ismcit.css" rel="stylesheet" type="text/css" id="styleId" />
		<meta
			name="Description"
			content="ISMICT 2024 will bring together top 
	  researchers from North America, Pacific areas, Europe and all around the 
	  world to exchange research results and address open issues in all aspects 
	  of Medical ICT."
		/>
		<meta
			name="Keywords"
			content="Biomedical, Bioinformatics, Molecular Communication and Computing, Medical, Healthcare."
		/>

		<script src="https://code.jquery.com/jquery-3.6.0.js"></script>
		<script>
			$(function () {
				$('#headerNav').load('./header.html');
				$('#twitter').load('./twitter.html');
				$('#footer').load('./footer.html');
			});
		</script>
		<style type="text/css">
			.tg1 .paperSessionName2 {
				font-family: Verdana, Arial, Heveltica;
				font-size: 14px;
				font-weight: bold;
				padding: 10px 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #3b3a30;
				color: #ffffff;
				vertical-align: top;
			}
			.tg1 .paperSessionChair2 {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: normal;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #b2c2bf;
				color: #000000;
				vertical-align: top;
			}
			.tg1 .paperSessionTime2 {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: normal;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #d0efea;
				color: #000000;
				vertical-align: top;
			}
			.tg1 .paperSessionTime3 {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: normal;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #b2c2bf;
				color: #000000;
				vertical-align: top;
			}
			.tg1 .paperInvitedTalk {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: bold;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #b2c2bf;
				color: #000000;
				vertical-align: top;
			}
			.tg1 .paperInvitedAuthorName {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: normal;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #f2f4ed;
				color: #000000;
				vertical-align: top;
			}
			.tg1 .paperInvitedTalkTime {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: normal;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #d0efea;
				color: #000000;
				vertical-align: top;
			}
			.tg1 .paperSessionPaperTitle {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: bold;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #f2f4ed;
				color: #000000;
				vertical-align: top;
			}
			.tg1 .paperSessionAuthorName {
				font-family: Verdana, Arial, Heveltica;
				font-size: 12px;
				font-weight: normal;
				padding: 5px;
				overflow: hidden;
				word-break: normal;
				background-color: #f2f4ed;
				color: #000000;
				vertical-align: top;
			}
		</style>
	</head>

	<body>
		<div id="headerNav"></div>
		<!--a name="table"-->
		<div id="centerBack">
			<div id="central">
				<div id="mainBody">
					<table class="tg1">
						<tbody>
							<tr>
								<td colspan="2" class="paperSessionName2" id="keynote1">
									Keynote 1:
								</td>
							</tr>
							<!-- <tr>
								<td colspan="2" class="paperSessionChair2">Session Chair: Josep Miquel Jornet (Northeastern University)</td>
				</tr> -->
							<tr>
								<td colspan="2" class="paperSessionTime2">
									Time: Wednesday, May XX - 09:00-10:00 (GMT)
								</td>
							</tr>
							<tr>
								<td class="photo">
									<img
										width="140"
										src="./pictures/user.svg"
										alt="User"
									/>
								</td>
								<td class="text">
									<p class="subsubtitle" align="left">User</p>
									<p class="plaintext">
										Professor<br />
										Biochemistry <br />
										University of ________<br />
									</p>
								</td>
							</tr>
						</tbody>
					</table>
					<!-- <p class="plaintext" align="justify">
					Digital twin technology has emerged as a powerful tool for creating virtual representations of real-world systems, allowing for the simulation, analysis, and optimization of these systems in a controlled environment. Life sciences is primed for digital twin technology, which can integrate diverse data sources and reconstruct comprehensive, dynamic models of biological systems. These medical digital twins can facilitate drug discovery, optimize treatment plans, and even replace traditional clinical trials with simulations on virtual patients. In this presentation, I will discuss our efforts toward the development of a general-purpose Immune Digital Twin. I will discuss its uses to better understand the dynamics of the immune system across different scales of biological organization. To exemplify our methods in the context of drug development, I will also share our very recent results of new antiviral combinatorial treatments for influenza identified from an underlying model of host-pathogen interactions between influenza and airway epithelial cells.
					</p>  -->

					<h3>About the Speaker:</h3>
					<p class="plaintext">Coming soon</p>

					<!-- <p class="plaintext" align="justify">
					<b>Dr. Helikar, PhD.,</b> is a Susan J Rosowski Professor in the Department of Biochemistry at the University of Nebraska-Lincoln and director of the Digital Twin Innovation Hub. Dr. Helikar holds courtesy appointments in the School of Computing and the Department of Pharmacology and Experimental Neuroscience. Dr. Helikar obtained his B.Sc. in Bioinformatics from the Computer Science Department at the University of Nebraska at Omaha in 2006, and his Ph.D. from the Department of Pathology and Microbiology at the University of Nebraska Medical Center in 2010. Dr. Helikar joined the University of Nebraska-Lincoln in 2013 as an Assistant Professor after completing a 3-year postdoctoral work in the Department of Mathematics at the University of Nebraska at Omaha.  Dr. Helikar’s research program has been funded by various U.S. federal agencies, including the National Science Foundation (NSF), National Institutes of Health (NIH), the Department of Defense (DoD), and the United States Department of Agriculture, as well as private sponsors, including Google. Dr. Helikar has co-authored nearly 80 manuscripts.
					</p> -->
					<br />

					[<a href="./program.html">Back to Program</a>] <br /><br />

					<table class="tg1">
						<tbody>
							<tr>
								<td colspan="2" class="paperSessionName2" id="keynote2">
									Keynote 2:
								</td>
							</tr>
							<!--tr>
								<td colspan="2" class="paperSessionChair2">Session Chair: Josep Miquel Jornet (Northeastern University)</td>
							</tr-->
							<tr>
								<td colspan="2" class="paperSessionTime2">
									Time: Wednesday, May XX - 14:00-15:00 (GMT)
								</td>
							</tr>
							<tr>
								<td class="photo">
									<img
										width="140"
										src="./pictures/user.svg"
										alt="User"
									/>
								</td>
								<td class="text">
									<p class="subsubtitle" align="left">User, Ph.D</p>
									<p class="plaintext">
										Professor<br />
										Biochemistry <br />
										University of ________<br />
									</p>
								</td>
							</tr>
						</tbody>
					</table>
					<!-- <p class="plaintext" align="justify">
					Gene delivery, or the transfer of protein-encoding nucleic acids into a cell, exhibits great potential for addressing questions and challenges in both research and medicine, with applications in functional genomics, tissue engineering, vaccines, and cell and gene therapies. Nonviral gene delivery systems offer a safer and more easily modified alternative to viral vectors, but often suffer from low delivery efficiencies and limited transgene expression (i.e., transfection). Investigation into nonviral gene delivery through computational modeling offers an alternative to focused empirical approaches, providing insight into complex cellular process behavior, as well as a means of testing hypotheses in a highly accessible context. We have developed a computational model of HeLa cell transfection based on telecommunications principles and queueing theory techniques, wherein the transfection process is represented as a packet-switched telecommunications network. The model was the first transfection model reported to incorporate the effects of both mitosis and transfection-induced toxicity, features critical for accurately recapitulating the full transfection process.  In addition, the model provided key insights into post-nuclear barriers that may prevent successful transfection. We have since translated the model to one representing transfection in human mesenchymal stem cells (hMSC), a clinically relevant cell type. Through model architecture modification and parameter optimization, we have demonstrated the roles of mitosis, toxicity, and post-nuclear events in defining hMSC transfection outcomes and donor-variability. Additionally, our hMSC transfection model suggests potential roles of two cellular processes not yet considered in transfection-related models: degradation of nuclear plasmid DNA following the onset of mitosis and spontaneous transcriptional suppression of transgene expression. This model provides a tool for probing the dynamic cellular processes involved in hMSC transfection, which can be used to improve nonviral gene delivery methods for therapeutic applications.
					</p>  -->

					<h3>About the Speaker:</h3>
					<p class="plaintext" align="justify"></p>
					<p class="plaintext">Coming soon</p>

					<!-- <b>Angela K. Pannier, PhD.,</b> is the Swarts Family Chair in Biological Systems Engineering, and a Professor of Biomedical Engineering, in the Department of Biological Systems Engineering at the University of Nebraska- Lincoln,  with a Courtesy Appointment in the Mary and Dick Holland Regenerative Medicine Program at the University of Nebraska Medical Center. Pannier has also served as a visiting scholar to the Institute of Polymer Research in Dresden, Germany. Pannier’s research is focused in engineering systems and biomaterials for cell and gene therapies. Pannier has received funding from the American Heart Association (Scientist Development Grant), National Science Foundation (CAREER Award and MRIs), NIH (Director’s New Innovator Award), the U.S. Department of Agriculture, and industry. Pannier is active in many societies (BMES, AIChE, ASGCT), serves as a reviewer for over 50 different journals and funding agencies (including NIH and NSF), serves as an Associate Editor for Science Advances, and serves on the editorial board for two journals: Experimental Biology and Medicine and Regenerative Medicine Frontiers. In 2019 Pannier was awarded the Presidential Early Career Award for Scientist and Engineers from the U.S. White House Office of Science and Technology Policy; Pannier is the first Nebraskan to receive this honor. 7 Pannier was awarded the 2020 Gamma Sigma Delta Outstanding Research Award and the 2020 College of Engineering Outstanding Research and Creative Activity award. Pannier has also been awarded the college-wide Outstanding Graduate Mentor Award (she has mentored over 20 graduate students) and the university-wide Outstanding Undergraduate Research Mentor Award (she has mentored over 50 undergraduates in her lab), as well as various teaching and service awards.  In 2020 Pannier was named a Biomedical Engineering Society Fellow and in 2022 was inducted into the College of Fellows in the American Institute of Medical and Biological Engineers. Pannier has also served on and led various departmental, college and university committees, and co-chaired the university-wide N2025 Strategic Planning team.  She has completed the Big Ten Academic Alliance Academic Leadership Program.  She holds a BS and MS in biological systems engineering from UNL and a PhD in biology from Northwestern University.
					</p> -->
					<br />

					[<a href="./program.html">Back to Program</a>] <br /><br />

					<table class="tg1">
						<tbody>
							<tr>
								<td colspan="2" class="paperSessionName2" id="keynote3">
									Keynote 3:
								</td>
							</tr>
							<!--tr>
								<td colspan="2" class="paperSessionChair2">Session Chair: Josep Miquel Jornet (Northeastern University)</td>
							</tr-->
							<tr>
								<td colspan="2" class="paperSessionTime2">
									Time: Thursday, May XX - 09:00-10:00 (GMT)
								</td>
							</tr>
							<tr>
								<td class="photo">
									<img
										width="140"
										src="./pictures/user.svg"
										alt="User"
									/>
								</td>
								<td class="text">
									<p class="subsubtitle" align="left">User, Ph.D</p>
									<p class="plaintext">
										Professor<br />
										Biochemistry <br />
										University of ________<br />
									</p>
								</td>
							</tr>
						</tbody>
					</table>
					<!-- <p class="plaintext" align="justify">
					This presentation will focus on fluorescent sensor array-enabled bacterial pathogen identification and classification. Timely and efficient identification of pathogenic microorganisms is of high importance for public health and safety. Most currently developed sensing methods are based on optical and/or electrochemical techniques, where selectivity is achieved by using antibodies, their fragments, natural or engineered peptides, or aptamers. Important limitations for using these elements as a part of the sensor are cost, stability, special handling, and a narrow range of analytes. Here a sensor array for multivariate analysis based on small molecule ratiometric fluorescent dyes will be described, including bacterial species and gram status identification. The adaptation to a paper-based platform will be described, which was the first step in developing a platform that could go beyond a traditional laboratory environment. Finally, our experience at implementing pattern analysis approaches will be discussed in the context of sensitivity and specificity of standard clinical approaches.
					</p>  -->

					<h3>About the Speaker:</h3>
					<p class="plaintext">Coming soon</p>

					<!-- <p class="plaintext" align="justify">
					<b>Dr. Aaron Mohs, PhD.,</b> is an associate professor in the Department of Pharmaceutical Sciences and is the Associate Dean for Research and Graduate Studies of the UNMC College of Pharmacy. Dr. Mohs is also a member of the Fred & Pamela Buffett Cancer Center. He received his B.A. in Chemistry from St. John’s University/College of St. Benedict (Collegeville, MN) and received his Ph.D. in Pharmaceutics and Pharmaceutical Chemistry at the University of Utah under Dr. Zheng-Rong Lu. Dr. Mohs then completed his postdoctoral fellowship in the joint Emory-Georgia Tech Department of Biomedical Engineering, as an Emory-Georgia Tech Center for Cancer Nanotechnology Excellence Distinguished Fellow with Dr. Shuming Nie. Dr. Mohs has an active, NIH-funded, collaborative research program on the development of fluorescent probes to guide the surgical resection of tumors, fluorescent sensor array technology that can rapidly detect and identify bacterial pathogens, and drug delivery systems to target multiple myeloma. Dr. Mohs is the immediate past-President of the UNMC Faculty Senate and recently served as the Director of the Pharmaceutical Sciences Graduate Program. Dr. Mohs is a recipient of the 2017 UNMC New Investigator award, 2019 UNMC Distinguished Scientist award, and, most recently, selected as a Fellow of the National Strategic Research Institute at the University of Nebraska. Prior to joining UNMC in 2015, Dr. Mohs was an assistant professor of Biomedical Engineering in the Wake Forest – Virginia Tech School of Biomedical Engineering and Sciences.
					</p> -->
					<br />

					[<a href="./program.html">Back to Program</a>] <br /><br />

					<table class="tg1">
						<tbody>
							<tr>
								<td colspan="2" class="paperSessionName2" id="invited1">
									Invited Talk:
								</td>
							</tr>
							<!--tr>
								<td colspan="2" class="paperSessionChair2">Session Chair: Josep Miquel Jornet (Northeastern University)</td>
							</tr-->
							<tr>
								<td colspan="2" class="paperSessionTime2">
									Time: Thursday, May XX - 09:00-10:00 (GMT)
								</td>
							</tr>
							<tr>
								<td class="photo">
									<img
										width="140"
										src="./pictures/user.svg"
										alt="User"
									/>
								</td>
								<td class="text">
									<p class="subsubtitle" align="left">User, Ph.D</p>
									<p class="plaintext">
										Professor<br />
										Biochemistry <br />
										University of ________<br />
									</p>
								</td>
							</tr>
						</tbody>
					</table>
					<!-- <p class="plaintext" align="justify">
					The talk covers a short introduction to the application of Machine Learning to the Genomic Data Science and an overview of several research projects in my laboratory. In addition, our experiences with building the computational ecosystem at UNMC and the development of cloud-based learning modules for analyzing Next-generation sequencing and other Omics data.
					</p>  -->

					<h3>About the Speaker:</h3>
					<p class="plaintext">Coming soon</p>

					<!-- <p class="plaintext" align="justify">
					<b>Dr. Babu Guda, PhD.,</b> is a Professor in Genetics, Cell Biology and Anatomy, and Assistant Dean for Research Development at the College of Medicine. He is also the founding director of the Bioinformatics and Systems Biology (BSB) Core, and the Center for Biomedical Informatics Research and Innovation at UNMC. Dr. Guda was trained in the interdisciplinary areas of molecular biology, computer science and computational biology, with over 25 years of experience in bioinformatics research. His current research projects are focused on developing novel computational methods and software for analyzing various types of ‘omics data including the next-generation sequencing data such as whole genome and Exome-seq, RNAseq, ATAC-seq, and Metagenomics. His group also employs the machine learning and deep learning methods to integrate and interrogate high-dimensional datasets for classification and prediction purposes. Dr. Guda helped develop several graduate programs and taught graduate courses in Bioinformatics and mentored dozens of graduate students, postdocs and junior faculty in the bioinformatics field. He served on numerous grant review panels of NSF, NIH, DoD, and MRC, and published about 150 peer-reviewed research articles that cover a wide range of topics related to computational biology, cancer genomics, machine learning and genomic data science.
					</p> -->
					<br />

					[<a href="./program.html">Back to Program</a>] <br /><br />
				</div>
				<div id="twitter"></div>
			</div>
			<div id="footer"></div>
		</div>

		<div id="Sep"></div>
	</body>
</html>
